helm/action/list.go

```diff
@@ -10,13 +10,15 @@ import (
 // List lists all of the local charts.
 func List(homedir string) {
 	md := filepath.Join(homedir, WorkspaceChartPath, "*")
+
 	charts, err := filepath.Glob(md)
-	if err != nil {
```

Comment:
just like to prevent accidental assignments instead of comparison.  it's a habit. i can back this out.